src/core/agent/services/multiSig.types.ts

```diff
@@ -1,20 +1,20 @@
 interface Aid {
```

Comment:
Can we use `HabState` or something like that from Signify-TS now and remove this interface? It would be good to check for these types and see if they are in Signify-TS now